Lars-02/Smoelenboek
# Fix register admin account

## PR to register an admin account

## Changes
- cleaned-up code in registration method
- added success message after creating account
- fixed failing registration of an admin

## Test
1. Login as an admin and go to a random profile
2. Go to registration page
3. Create an admin
4. Logout
5. Log in with the newly created admin
6. Display a profile and check whether or not the edit page is available for the user

## Finishing tests
Test should pass
